Arbie/Contracts/tokens.py

```diff
@@ -1,9 +1,42 @@
 """Utility functions for interacting with Tokens."""
 
-from Arbie.Contracts import Contract
+from Arbie import BigNumber
+from Arbie.Actions.arbitrage import Token
+from Arbie.Contracts import Address, Contract
 
 
 class GenericToken(Contract):
```

Comment:
Make task of regarding GenericToken shot comings.

* All tokens are named BNB
* We need ABI for non BNB ERC20 token.
* We should seperate between BNB token and GenericToken

Reply:
This actually works good enough for now! Tested on real ERC20 tokens.